assignment/tool/src/webapp/vm/assignment/chef_assignments_student_view_submission.vm

```diff
@@ -686,8 +693,15 @@ $(document).ready(function(){
 							<img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" class="assignmentRosette"/>
 							<em>
 								$plagiarismNote
-								#if ($plagiarismFileTypes)
-									$plagiarismFileTypes
+								#if ($plagiarismFileTypes || $plagiarismFileTypes)
```

Comment:
And here

Reply:
Thanks, I did it on the wrong branch and not fixed it in here.